src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_FrameworkElement.cs

```diff
@@ -14,27 +14,18 @@
 using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Controls.Primitives;
 using Private.Infrastructure;
+using MUXControlsTestApp.Utilities;
 
 namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls
 {
 	[TestClass]
 	public class Given_FrameworkElement
 	{
-		private async Task Dispatch(DispatchedHandler p)
-		{
-#if !NETFX_CORE
-			await CoreApplication.GetCurrentView().Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, p);
-#else
-			await CoreApplication.MainView.Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, p);
-#endif
-		}
-
 #if __WASM__
 		// TODO Android does not handle measure invalidation properly
 		[TestMethod]
-		public async Task When_Measure_Once()
-		{
-			await Dispatch(() =>
+		public Task When_Measure_Once() =>
+			RunOnUIThread.Execute(() =>
```

Comment:
The is a nice new attribute from @davidjohnoliver for that :)

Reply:
I did this before ;-)